app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -107,6 +107,26 @@ def pvm_instances
         :value        => instance.proc_type,
         :read_only    => true
       )
+
+      # saving pin_policy
+      persister.vms_and_templates_advanced_settings.build(
+        :resource     => ps_vmi,
+        :name         => 'pin_policy',
+        :display_name => _('Pin Policy'),
+        :description  => _('VM Pin Policy'),
+        :value        => instance.pin_policy,
+        :read_only    => true
+      )
+
+      ldesc = software_licenses_description(instance.software_licenses)
+      ldesc.empty? || persister.vms_and_templates_advanced_settings.build(
```

Comment:
For some reason this reads strange to me, I think because it's a negative followed by a positive and using `||` for control flow.  I'm fine with the latter, but I wonder if this is more readable:

```suggestion
      ldesc.present? && persister.vms_and_templates_advanced_settings.build(
```

Reply:
yes, it is definitely more readable. I was not sure if ldesc.present? will take care of the blank string.